modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -3348,10 +3348,21 @@ def broadcast_apply_full_axis(
         PandasDataframe
             New Modin DataFrame.
         """
+
+        def get_partitions(df):
+            """Deal with the corner case if the "other" dataframe has no partitions."""
+            if df._partitions.size > 0:
+                return df._partitions
+            else:
+                empty_partition = self._partition_mgr_cls.create_partition_from_data(
+                    pandas.DataFrame(index=df.index, columns=df.columns)
+                )
```

Comment:
```suggestion
                return np.array([[self._partition_mgr_cls._partition_class.put(
                    pandas.DataFrame(index=df.index, columns=df.columns)
                ]])
```

Reply:
committed this, for now but I think the readability would be better with using `empty_partition` or `empty_data_partition` as a variable and returning the variable. Is  it a better practice to not create a variable and return directly as this would be slightly confusing?